resources/lang/en/filament-value-range-filter.php

```diff
@@ -6,5 +6,9 @@
     'range.options.between' => 'is between',
     'range.options.greater_than' => 'is greater than',
     'range.options.less_than' => 'is less than',
+    'range.indicator.equal' => ':label is equal to :value',
+    'range.indicator.between' => ':label is between :formValue and :toValue',
```

Comment:
@buddhaCode  Awesome! Thank you for all contributions :)

Could you fix the typo `:formValue` to  `:fromValue` in
`':label is between :formValue and :toValue',`

Reply:
whoops. sure thing :)

Reply:
Thank you :)